### PR TITLE
signup flow

### PR DESCRIPTION
**Pull request by bariscc**
- [x] Moved to resolver.js
- [x] Implemented validations
- [x] Return token
- [ ] modify signup form to make graphql request (frontend)
- [ ] add token to graphql requests (frontend)

### CODE REVIEW

**Comment by bariscc** [comment]
@SelmanKahya Its good to go, I'll implement the frontend part soon. There are no breaking changes.

File: src/constants.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export const RECAPTCHA_API_KEY = '6Lfxa3kUAAAAACOcWknFxfEbQWp9-x439LVBtPMK'; // bariscc
```
Comment (SelmanKahya) [inline src/constants.js]: we can leave a TODO here, needs to come from env, and need to get a key that will fulfill our request load (i havent checked pricing, or number of reqs/sec allowed)

Comment (bariscc) [reply src/constants.js]: true, we should keep that in env. also we can use the test keys until launch. 
about the request load, afaik no limits are enforced.

Comment (SelmanKahya) [reply src/constants.js]: 👍👍 feel free to leave a todo comment in your next pr

**Pull request merged**